### ISSUE TITLE
Manual annotation file is missing

### ISSUE THREAD

**Issue by engrodawood** [huiqu18/GeneMutationFromHE#1]
Thanks for the amazing piece of work and well structure code. I am trying to replicate your results but when I am running the fuse-clustering script the manual annotation file is missing. Can you please upload the missing file? 

Many thanks.

**Comment by monjoybme** [comment]
@engrodawood  Are you looking for this [code](https://github.com/huiqu18/GeneMutationFromHE/blob/main/code_data_processing/2-4.fuse_clustering_and_manual.py)?

**Comment by engrodawood** [comment]
Yes. But the manual annotation file is missing.

**Comment by monjoybme** [comment]
@huiqu18 Could you please tell us if you can share some information on manual annotation file?

**Comment by huiqu18** [comment]
Sorry for the late reply. I have graduated and have no access to the server that contains the original data. We should have those files. I will contact the other authors regarding this issue.

**Comment by engrodawood** [comment]
> @huiqu18 Could you please tell us if you can share some information on manual annotation file?

Yes. By manual annotation we means the tumor region annotated by pathologist.

**Comment by engrodawood** [comment]
> Sorry for the late reply. I have graduated and have no access to the server that contains the original data. We should have those files. I will contact the other authors regarding this issue.

Thanks for responding. Looking forward for the good news regarding annotation data🙂.
Thanks

**Comment by huiqu18** [comment]
Hi, the manual annotation files can be downloaded here: [manual_annotation](https://drive.google.com/file/d/1X39LMGZG5yJzXce-R4xwC-dUfiHFv4aw/view?usp=sharing)
The trained models of all results can be downloaded here: [trained_models](https://drive.google.com/file/d/1X51jIPpWugOyAOLj3muJHbo57F39rFpF/view?usp=sharing)

**Comment by engrodawood** [comment]
Thank @huiqu18 for the amazing work and for sharing the annotations and trained models.
